src/apscheduler/_schedulers/async_.py

```diff
@@ -529,6 +532,29 @@ async def remove_schedule(self, id: str) -> None:
         self._check_initialized()
         await self.data_store.remove_schedules({id})
 
+    async def pause_schedule(self, id: str) -> None:
+        """Pause the specified schedule."""
+        self._check_initialized()
+        await self.data_store.pause_schedules({id})
```

Comment:
This code should be in the scheduler (i.e. this method) if it doesn't require any additional functionality from the data store.

Reply:
Good call. Addressed in 61f7051. One consequence of this is that it no longer makes sense to have datastore tests for pausing and unpausing, so those tests have been removed.